### DISCUSSION TITLE
rename mingw's i386 target to i686

### DISCUSSION THREAD

**Discussion by waruqi** [xmake-io/xmake#5906] [Polls]
https://github.com/xmake-io/xmake/discussions/5904

**Comment by TurnOffNOD** [comment]
i686 means that compiled binary should run at least 686 architecture, and means it drops out suport of architecture older than 686.

i386 means that it still supports architecture older than 686.

**Comment by waruqi** [comment]
I added a poll option, `x86_64, i386, i686`

**Comment by ifarbod** [comment]
I'd say **add** and **rename** to i686 and have backward compatibility for i386, x86
